MDM migration: Update FileVault key escrow instructions (#47807)
fleetdm/fleet: articles/mdm-migration.md

_Available in Fleet Premium_

> When migrating hosts that enroll manually from a previous MDM, end users must log out of their device to escrow FileVault keys to Fleet. The **My device** page in Fleet Desktop will present users with instructions on how to reset their key.

Author: ```suggestion
When migrating hosts via manual enrollment profile, end users must log out of their device to escrow FileVault keys to Fleet. The **My device** page in Fleet Desktop will present users with instructions on how to reset their key.
```

To start, [enforce FileVault disk encryption](https://fleetdm.com/guides/enforce-disk-encryption) in Fleet.

After turning on disk encryption in Fleet, share [these guided instructions](#how-to-turn-on-disk-encryption) with your end users.

> For hosts that enroll automatically, via Apple Business, end users don't need to take action. Fleet automatically escrows the FileVault key.

Author: ```suggestion
For hosts that enroll via Apple Business, end users don't need to take action. Fleet automatically escrows the FileVault key on the next host vitals refetch.
```
